CMakeLists.txt

```diff
@@ -28,6 +28,12 @@ target_link_libraries(project1 project1_lib)
 # Testing
 ################################
 if (test)
+  # only for apple computer
```

Comment:
i got following error when i compiling on my Macbook. 

```cpp
fatal error: 'tr1/tuple' file not found
#include <tr1/tuple> // NOLINT
```
after Google i found that,

> On OS X, only in case you select Google Test v1.6.0, you’ll need to add the following lines to your project’s CMakeLists.txt:

```cmake
if (APPLE)
    add_definitions(-DGTEST_USE_OWN_TR1_TUPLE)
    add_definitions(-D__GLIBCXX__)
endif (APPLE)
```
so  that's why i add this